onnxruntime/python/tools/symbolic_shape_infer.py

```diff
@@ -225,6 +225,7 @@ def __init__(self, int_max, auto_merge, guess_output_rank, verbose, prefix=""):
             "SimplifiedLayerNormalization": self._infer_LayerNormalization,
             "SkipLayerNormalization": self._infer_SkipLayerNormalization,
             "SkipSimplifiedLayerNormalization": self._infer_SkipLayerNormalization,
+            "MatMulNBits": self._infer_MatMulNBits,
```

Comment:
Can you insert `MatMulNBits` into the dictionary in alphabetical order? Can you also fix the `SparseAttention` and `SkipGroupNorm` entries?